modules/oxide/tasks/less-task.js

```diff
@@ -0,0 +1,51 @@
+const less = require('less');
+const fs = require('node:fs');
+const path = require('path');
+const LessPluginPresetEnv = require('less-plugin-preset-env');
+
+const presetEnv = new LessPluginPresetEnv({
+  browsers: ['last 2 Safari versions', 'iOS 14.0', 'last 2 Chrome versions', 'Firefox ESR'],
```

Comment:
```suggestion
  browsers: ['last 2 Safari versions', 'last 2 iOS major versions', 'last 2 Chrome versions', 'Firefox ESR'],
```
I was wondering if this was possible, looks like it is:
https://github.com/browserslist/browserslist#full-list

Unfortunately it doesn't seem to be capable of "chrome extended stable", although that changes frequently enough that we should be ok.